tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -108,11 +79,44 @@ extension XCScheme {
                 launchActionInfo: launchActionInfo,
                 otherPreActions: otherPreActions
             )
+            launchActionHasRunnable = true
         } else {
             launchAction = .init(
                 runnable: nil,
                 buildConfiguration: schemeInfo.defaultBuildConfigurationName
             )
+            launchActionHasRunnable = false
+        }
+
+        let testAction: XCScheme.TestAction?
+        if let testActionInfo = schemeInfo.testActionInfo {
+            // TODO: Make this similar to `initBazelBuildOutputGroupsFile()`,
+            // instead of `otherPreActions`
+            var otherPreActions: [XCScheme.ExecutionAction] = []
+            if let aTargetInfo = testActionInfo.targetInfos
+                .lazy
+                .sortedLocalizedStandard(\.pbxTarget.name)
+                .first
+            {
+                otherPreActions.append(
+                    .createPreActionScript(
+                        title: "Update .lldbinit",
+                        scriptText: ExecutionAction.createLLDBInitScript,
+                        buildableReference: aTargetInfo.buildableReference
+                    )
+                )
+            }
+            testAction = try .init(
+                buildMode: buildMode,
+                testActionInfo: testActionInfo,
+                launchActionHasRunnable: launchActionHasRunnable,
```

Comment:
Would it be a bit simpler just directly testing the actual value?
```suggestion
                launchActionHasRunnable: launchAction.runnable != nil,
```

Reply:
Yeah, probably.